vignettes/vrd_vignette.Rmd

```diff
@@ -0,0 +1,259 @@
+
+**Example:** If you wanted to search how many items were recalled from the start of the database to the manufacturing year of 2010 you could use the following.
+```{r count_recall_by_years, eval=FALSE}
+count_recall_by_years(start_year = 2010)
+#>   Result Count
+#> 1        32030
+```
+
+#### recall_by_number()
+
+Used to retrieve a data frame listing of recalls between for the provided recall numbers. It has **three** inputs:
+
+- recall_number: A seven digit number of an item recall. Can be a list.
+- limit: An integer that specifies how recalls you want to return. If left empty defaults to 25.
+- api_key: A string of the api key needed to access the database. If left empty will error and not provide requested data frame.
+
+**Example:** If you knew the recall numbers of you wanted to search for you could use the following.
+```{r recall_by_number, eval=FALSE}
+recall_by_number(c(1977044,1977045, 1977046))
+#>   Recall number Manufacturer Name    Model name  Make name Year Recall date
+#> 1       1977044              FORD         PINTO       FORD 1977  1977-08-09
+#> 2       1977045   CLARKE SIMPKINS    850 LEMANS MOTO GUZZI 1977  1977-08-10
+#> 3       1977045   CLARKE SIMPKINS 1000 CIVILIAN MOTO GUZZI 1977  1977-08-10
+#> 4       1977046    GENERAL MOTORS      C SERIES        GMC 1977  1977-08-16
+#> 5       1977046    GENERAL MOTORS      C SERIES  CHEVROLET 1977  1977-08-16
+```
+
+
+#### recall_details()
+
+Used to retrieve a data frame listing summary data of recall(s) involving an item between for the provided recall number(s). It has **three** inputs:
+
+- recall_number: A seven digit number of the item recall. Can be a list.
+- limit: An integer that specifies how recalls you want to return. If left empty defaults to 25.
+- api_key: A string of the api key needed to access the database. If left empty will error and not provide requested data frame.
+
+**Example:** If you wanted to search how many items were recalled from the start of the database to the manufacturing year of 2010 you could use the following.
+```{r recall_details, eval=FALSE}
+recall_details(1977044)
+#>   RECALL_NUMBER_NUM MANUFACTURER_RECALL_NO_TXT CATEGORY_ETXT CATEGORY_FTXT MODEL_NAME_NM MAKE_NAME_NM UNIT_AFFECTED_NBR SYSTEM_TYPE_ETXT SYSTEM_TYPE_FTXT
+#> 1           1977044                                      Car       Voiture         PINTO         FORD                28      Not Entered       Non Saisie
+#>   NOTIFICATION_TYPE_ETXT NOTIFICATION_TYPE_FTXT
+#> 1            Not Entered             Non Saisie
+#>                                                                                                                                               COMMENT_ETXT
+#> 1 VEHICLES WERE BUILT WITH INCORRECT NUTS ATTACHING THE REAR BUMPER LEFT ISOLATOR TO THE FRAME.  THESE NUTS COULD CONTACT AND PUNCTURE THE FUEL TANK. \r\n
+#>                COMMENT_FTXT DATE_YEAR_CD RECALL_DATE_DTE
+#> 1 Translation not available         1977      1977-08-09
+```
+
```

Comment:
maybe display this data different so that it's not in this format

Reply:
It seems like this commented out format is standard in vignettes so that your running as little code as possible.  I'm still trying to figure out if its typical to run plot code at all or if they just paste in the respective plot.